common/app/templates/inlineJS/blocking/loadApp.scala.js

```diff
@@ -26,6 +26,18 @@ function guardianPolyfilled() {
     } catch (e) {};
 }
 
+function shouldServeLotame() {
+    try {    
+        var geo = JSON.parse(window.localStorage.getItem("gu.geolocation")).value;
```

Comment:
Is this because the geo module hasn't loaded yet?

Reply:
@kelvin-chappell Exactly. None of our application has loaded; not even standard.

The other thing to note here is that `gu.geolocation` also contains an `expiry`; I have decided to ignore it. It will get refreshed anyway if it has expired.